payments/models.py

```diff
@@ -39,6 +40,63 @@ def __setattr__(self, key, value):
         return None
 
 
+class BaseSubscription(models.Model):
+    token = models.CharField(
+        _("subscription token/id"),
+        help_text=_("Token/id used to identify subscription by provider"),
+        max_length=255,
+        default=None,
+        null=True,
+        blank=True,
+    )
```

Comment:
Any particular reason for allowing blank/null values? I mean, allowing this usually implies a lot of tedious None-checking in the code.

Reply:
But could we expect, that all providers will have tokens/IDs?